frontend/src/main/web/index.js

```diff
@@ -0,0 +1,49 @@
+/*
+This node script will loop through all sibling directories and invoke npm command on each one of them.
+To use this script, run: node index.js install or node index.js run build
```

Comment:
This file is not used by maven. I can remove it if you like. But I thought it may come handy when running node/npm directly during development.